### PR TITLE
update default game script and doc

### PR DESCRIPTION
**Pull request by gdevillele**

### CODE REVIEW

File: game-scripts/default.lua [hunk 1]
```diff
@@ -63,26 +62,21 @@ Config.items = {nil, R.pen, R.pickaxe}
...
+Local.dropAboveCenter = function()
     Player.Position = { Map.Width * 0.5, Map.Height  + 10, Map.Depth * 0.5 }
     Player.Rotation = { 0, 0, 0 }
     Player.Velocity = { 0, 0, 0 }
 end
 
--- We can call it right now!
--- Function calls at the root level in the script
--- are executed when the script is loaded.
-dropAboveCenter()
```
Comment (aduermael) [inline game-scripts/default.lua]: We could keep it in place. 
It's possible to call functions here.

File: game-scripts/default.lua [hunk 2]
```diff
@@ -63,26 +62,21 @@ Config.items = {nil, R.pen, R.pickaxe}
...
-
--- Define primary action function
+-- Define primary action function for the local Player
 -- (left click on desktop, primary action button on mobile)
+-- Note that "Player" is a shortcut for "Local.Player"
```
Comment (aduermael) [inline game-scripts/default.lua]: Maybe that should be mentioned earlier. 
`Player` is used in `dropAboveCenter`

File: game-scripts/dontfall.lua [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+-- in the game. 
+-- NOTE: players can jump differently based on the item
+-- they're holding for example.
+Local.Player.Jump = function (player)
+    -- Test if player is on ground before changing velocity,
+    -- otherwise, player could jump while in the air. :D
+    if player.IsOnGround then
+      player.Velocity.Y = Config.DefaultJumpStrength
+    end
+end
```
Comment (aduermael) [inline game-scripts/dontfall.lua]: We should not be able to jump in `Don't fall`

Comment (gdevillele) [reply game-scripts/dontfall.lua]: Thank you ! 😲

**Pull request merged**